editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -654,6 +655,74 @@ export const printCode = memoize(printCodeImpl, {
   maxSize: 1,
 })
 
+function isVariableDeclaration(line: string): boolean {
+  const withoutExport = line.startsWith('export ') ? line.slice(7) : line
+  return (
+    withoutExport.startsWith('var') ||
+    withoutExport.startsWith('let') ||
+    withoutExport.startsWith('const')
+  )
+}
+
+function isFunctionDeclaration(line: string): boolean {
+  const withoutExport = line.startsWith('export ') ? line.slice(7) : line
+  const withoutDefault = withoutExport.startsWith('default ')
+    ? withoutExport.slice(8)
+    : withoutExport
+  return withoutDefault.startsWith('function')
+}
+
+function createBlanklineTracker() {
```

Comment:
Would it be possible to have a return type on this function please?

Reply:
Done